extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -233,6 +233,7 @@ export const popupMessageListener = (request: Request) => {
     let keyStore = { id: "" };
 
     try {
+      // const key = await keyManager.loadAllKeyIds();
```

Comment:
Watch out for merging this. Let's just comment out any flakey tests and we can fix those in a follow up PR

Reply:
results.length